### PR TITLE
Fix test case recorder

### PR DESCRIPTION
**Pull request by pokey**
The test case recorder broke in #1281 due to the fact we're now launching extension in subdirectory called `dist` rather than top-level repo.  This PR fixes it

## Checklist

- [x] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)
- [ ] I have updated the [docs](https://github.com/cursorless-dev/cursorless/tree/main/docs) and [cheatsheet](https://github.com/cursorless-dev/cursorless/tree/main/cursorless-talon/src/cheatsheet)
- [ ] I have not broken the cheatsheet

### CODE REVIEW

**Review by AndreasArvidsson — DISMISSED** [review]

**Comment by pokey** [comment]
For some reason that I can't fathom, we're getting a CI failure on Windows because even though the test case recorder is _correctly_ outputting a file with LF endings there,  the _expected_ content has CRLF endings. This is quite strange, because we're just reading a gold file that has LF endings.  It's almost like GitHub action runner is turning on `git config core.autocrlf` or something 🤔 cc/ @AndreasArvidsson @auscompgeek any ideas?

File: tsconfig.base.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "moduleResolution": "nodenext",
     "moduleDetection": "force",
     "target": "es6",
-    "lib": ["es2020"],
+    "lib": ["es2021"],
```
Comment (pokey) [inline tsconfig.base.json:7]: needed to bump this to use `.replaceAll`.  Seemed to work locally; we'll see if it passes on our legacy VSCode test

Comment (pokey) [reply tsconfig.base.json:7]: Looks like it's fine even on legacy VSCode

**Comment by pokey** [comment]
> It's probably fairly likely this is enabled in the global config

but how / why?  I don't see anywhere that this is something GitHub does in their runners 🤔

**Comment by auscompgeek** [comment]
Maybe we should try setting text=lf in .gitattributes 🤔

**Comment by pokey** [comment]
I'm hesitant to mess with that setting, as I don't fully understand it. It would be great to get this PR in, as the test case recorder is currently broken on main. Shall I file a follow-up to look into line endings?

**Comment by Leeft** [comment]
> Maybe we should try setting text=lf in .gitattributes 🤔

Yeah, that seems like a good idea: https://dev.to/deadlybyte/please-add-gitattributes-to-your-git-repository-1jld

**Comment by pokey** [comment]
Ok I filed #1348 to handle line endings properly

**Pull request merged**